### PR TITLE
Post: Release v2.6.0

### PR DESCRIPTION
**Pull request by agoscinski**

### CODE REVIEW

File: docs/news/posts/2024-07-02-release-2.6.0.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+category: Releases
+tags: aiida-core
+date: 2024-07-02
+---
+
+# AiiDA v2.6.0 released
+
+We just released the latest version of AiiDA. This minor release comes with a number of features that are focused on user friendliness of the CLI and the API. The [changelog](https://github.com/aiidateam/aiida-core/blob/v2.6.0/CHANGELOG.md) contains a list of all changes including a nice overview of some of the new features.
+
+A lot of work has especially gone in making it easier to install AiiDA and try it out without having to install services like PostgreSQL and RabbitMQ. It is now possible to operate AiiDA without RabbitMQ, at the cost of losing daemon functionality. The new command verdi presto allows setting up a complete and ready-to-go profile with a single command without any configuration. The installation [documentation](https://aiida.readthedocs.io/projects/aiida-core/en/stable/installation/index.html) has seen a complete overhaul.
```
Comment (edan-bainglass) [inline docs/news/posts/2024-07-02-release-2.6.0.md:12]: `verdi presto` to distinguish the command from surrounding text

Comment (edan-bainglass) [reply docs/news/posts/2024-07-02-release-2.6.0.md:12]: that is, add tick marks :)

**Review by edan-bainglass — APPROVED** [review]
LGTM (with one minor suggestion)

File: docs/news/posts/2024-07-02-release-2.6.0.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+blogpost: true
+category: Releases
+tags: aiida-core
+date: 2024-07-02
```
Comment (GeigerJ2) [inline docs/news/posts/2024-07-02-release-2.6.0.md:5]: We released on 2024-07-01, though :p

**Review by GeigerJ2 — CHANGES_REQUESTED** [review]

File: docs/news/posts/2024-07-02-release-2.6.0.md (hunk 2 above)
Comment (agoscinski) [reply docs/news/posts/2024-07-02-release-2.6.0.md:5]:
```suggestion
date: 2024-07-01
```

File: docs/news/posts/2024-07-02-release-2.6.0.md (hunk 1 above)
Comment (agoscinski) [reply docs/news/posts/2024-07-02-release-2.6.0.md:12]:
```suggestion
A lot of work has especially gone in making it easier to install AiiDA and try it out without having to install services like PostgreSQL and RabbitMQ. It is now possible to operate AiiDA without RabbitMQ, at the cost of losing daemon functionality. The new command `verdi presto` allows setting up a complete and ready-to-go profile with a single command without any configuration. The installation [documentation](https://aiida.readthedocs.io/projects/aiida-core/en/stable/installation/index.html) has seen a complete overhaul.
```

**Comment by agoscinski** [comment]
Needs a merge before
https://github.com/aiidateam/aiida-website/pull/70

**Review by GeigerJ2 — COMMENTED** [review]
CI still failing 😢

File: docs/conf.py [hunk 3]
```diff
@@ -151,6 +151,9 @@
     r"http://dx.doi.org/10.5075/epfl-thesis-7179",  # EPFL blocks repeating requests
     r"http://qe2019.ijs.si/index.html",  # event page does not exist anymore
 ]
+linkcheck_anchors_ignore_for_url = [
+    r"https://github.com/aiidateam/aiida-core/blob/v2.6.0/CHANGELOG.md/.*",  # linkcheck cannot figure out the anchor but it works
```
Comment (agoscinski) [inline docs/conf.py:155]:
```suggestion
    r"https://github\.com/aiidateam/aiida-core/blob/v2.6.0/CHANGELOG.md/.*",  # linkcheck cannot figure out the anchor but it works
```

File: docs/news/posts/2024-07-01-release-2.6.0.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+A lot of work has especially gone in making it easier to install AiiDA and try it out without having to install services like PostgreSQL and RabbitMQ. It is now possible to operate AiiDA without RabbitMQ, at the cost of losing daemon functionality. The new command `verdi presto` allows setting up a complete and ready-to-go profile with a single command without any configuration. The installation [documentation](https://aiida.readthedocs.io/projects/aiida-core/en/stable/installation/index.html) has seen a complete overhaul.
+
+The caching mechanism has received a number of improvements guaranteeing even greater savings of computational time. For existing calculations to be valid cache sources in the new version, their hash has to be regenerated (see [Improvements and changes to caching](https://github.com/aiidateam/aiida-core/blob/v2.6.0/CHANGELOG.md/#improvements-and-changes-to-caching) for details).
```
Comment (agoscinski) [inline docs/news/posts/2024-07-01-release-2.6.0.md:14]:
```suggestion
The caching mechanism has received a number of improvements guaranteeing even greater savings of computational time. For existing calculations to be valid cache sources in the new version, their hash has to be regenerated (see [Improvements and changes to caching](https://aiida.readthedocs.io/projects/aiida-core/en/stable/reference/_changelog.html#improved-test-fixtures-without-services) for details).
```

File: docs/conf.py [hunk 5]
```diff
@@ -151,6 +146,9 @@
     r"http://dx.doi.org/10.5075/epfl-thesis-7179",  # EPFL blocks repeating requests
     r"http://qe2019.ijs.si/index.html",  # event page does not exist anymore
 ]
+linkcheck_anchors_ignore_for_url = [
+    r"https://github\.com/aiidateam/aiida-core/blob/v2\.6\.0/CHANGELOG.md/.*",  # linkcheck cannot figure out the anchor but it works
+]
```
Comment (agoscinski) [inline docs/conf.py:151]:
```suggestion
```

File: docs/news/posts/2024-07-01-release-2.6.0.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+---
+blogpost: true
+category: Releases
+tags: aiida-core
+date: 2024-07-01
+---
+
+# AiiDA v2.6.0 released
+
+We just released the latest version of AiiDA. This minor release comes with a number of features that are focused on user friendliness of the CLI and the API. The [changelog](https://github.com/aiidateam/aiida-core/blob/v2.6.0/CHANGELOG.md) contains a list of all changes including a nice overview of some of the new features.
```
Comment (agoscinski) [inline docs/news/posts/2024-07-01-release-2.6.0.md:10]:
```suggestion
We just released the latest version of AiiDA. This minor release comes with a number of features that are focused on user friendliness of the CLI and the API. The [changelog](https://aiida.readthedocs.io/projects/aiida-core/en/stable/reference/_changelog.html#v2-6-0-2024-07-01) contains a list of all changes including a nice overview of some of the new features.
```

**Review by edan-bainglass — APPROVED** [review]
LGTM!

**Pull request merged**